src/java.base/share/classes/module-info.java

```diff
@@ -199,6 +199,7 @@
         jdk.attach,
         jdk.charsets,
         jdk.compiler,
+        jdk.crypto.cryptoki,
```

Comment:
At some point I'd like to see this re-visited so we can avoid exporting this package to modules defined to the platform class loader.